src/python_testing/TC_IDM_4_2.py

```diff
@@ -42,6 +44,23 @@
 https://github.com/CHIP-Specifications/chip-test-plans/blob/master/src/interactiondatamodel.adoc#tc-idm-4-2-subscription-response-messages-from-dut-test-cases-dut_server
 '''
 
+class AttributeChangeCallback:
```

Comment:
If you can move this into the base, that would be good - we have stuff there for events already, but I'm worried we're starting to get a lot of copy/paste for subscriptions.

Reply:
Done